storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -1,6 +1,6 @@
 /******************************************************
 hot backup tool for InnoDB
-(c) 2009-2015 Percona LLC and/or its affiliates
+(c) 2009-2021 Percona LLC and/or its affiliates
```

Comment:
Thanks for reminding me :) we should write a git commit hook that verifies this.
ie. list of modified files should have current year